### ISSUE TITLE
Unexpected chars are displayed on Manage my Submission in `ar`

### ISSUE THREAD

**Issue by ioanarusiczki** [mozilla/addons-server#18937]
On Manage my Submission page with `ar`  -> https://addons-dev.allizom.org/ar/developers/ at the bottom after the page counting there's a line that displays unexpected chars:

![ar](https://user-images.githubusercontent.com/33448286/156745029-efe1324c-9790-4275-8f6c-aefe9d7e6e66.png)

**Comment by eviljeff** [comment]
it's https://addons-dev.allizom.org/ar/developers/addons I presume.

Fixed in https://github.com/mozilla/addons-server/commit/a4be5310f474ed91fac7a5de1a107dc2d2e17c82 - the substitution tokens (and html entity) in the localized string got mangled - e.g. `%(begin)s` ended up as ` ٪ (start) s `.  I fixed the two other strings that seems to have swapped out the `%` for `٪`, along with the characters for `&` and `;`.

cc @peiying2
